src/components/Contacts/ContactRow.stories.tsx

```diff
@@ -1,8 +1,13 @@
 import React from 'react';
 import { Story, Meta } from '@storybook/react';
 import { withDesign } from 'storybook-addon-designs';
+
+import { gqlMock } from '../../../__tests__/util/graphqlMocking';
 import { ContactRow } from './ContactRow';
-import { ContactRowFragment } from './ContactRow.generated';
+import {
+  ContactRowFragment,
+  ContactRowFragmentDoc,
```

Comment:
So does this file get generated by graphqlMocking file?

Reply:
No, this is from codegen. We write the `.graphql` file and it generates an export with the contents of the `.graphql` file wrapped in `gql` from `graphql-tag`.
`src/components/Contacts/ContactRow.generated.ts` contains:
```ts
export const ContactRowFragmentDoc = gql`
  fragment ContactRow on Contact {
    ...
  }
`;
```